README.txt

```diff
@@ -8,33 +8,74 @@ Stable tag: 1.3.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
-An implementation of PHP Console for WordPress. Use Chrome Dev Tools to debug your WordPress installation!
+An implementation of PHP Console as a WordPress plugin.
+Use Chrome Dev Tools to debug your WordPress installation!
 
 == Description ==
 
-WP PHP Console allows you to handle WordPress PHP errors & exceptions, dump variables, execute PHP code remotely and many other things using Google Chrome extension PHP Console and PhpConsole server library.
```

Comment:
Both "PHP Console" and "PhpConsole" were used to denote the same. So I chose PHP Console in the documentation.